stable/gemini/values.yaml

```diff
@@ -0,0 +1,11 @@
+image:
+  tag: build_134 # TODO: replace with a version tag
```

Comment:
This should probably be done before we release

Reply:
👍 yeah, I'm probably going to hold off on merging until I get a versioned tag in